src/index.js

```diff
@@ -33,7 +34,7 @@ const Builder = ({
   const [currentPage, setCurrentPage] = useState(initial)
 
   useEffect(() => {
-    setCurrentPage(initial)
+    setCurrentPage(initial || 0)
```

Comment:
Explain please why we need this condition here ?

Reply:
If the initial data is not defined.